Cargo.toml

```diff
@@ -12,3 +12,7 @@ reqwest = { version = "0.11.13", features = ["blocking", "json"] }
 thiserror = "1.0.37"
 toml = "0.5.9"
 url = "2.3.1"
+
+[dev-dependencies]
+assert_fs = "1"
+predicates = "2.1"
```

Comment:
Were you planning to use this..?

Reply:
heh i was but didn't clean up the cargotoml, good catch